service.subtitles.napisy24pl/resources/lib/__init__.py

```diff
@@ -0,0 +1 @@
+# Dummy file to make this directory a package.
```

Comment:
in python3 I doubt you need this file anymore

Reply:
Should i remove it?

Reply:
Try and see :)